modules/system/aoe.js

```diff
@@ -36,7 +36,11 @@ export default class AbilityTemplate extends MeasuredTemplate {
     {
       if (aoeString.toLowerCase().includes(game.i18n.localize("AoE").toLowerCase()))
         aoeString = aoeString.substring(aoeString.indexOf("(")+1, aoeString.length-1)
-      
+
+      let test = game.messages.get(messageId).getTest();
```

Comment:
This is clunky, the point of the diameter parameter is to be able to configure this behavior. The solution here should be to call the function with the correct diameter argument

Reply:
No clue about this, so you can close this one if you want.